FlowCrypt/Common/Services/Encrypted Storage/KeyChainService.swift

```diff
@@ -82,4 +78,39 @@ struct KeyChainService: KeyChainServiceType {
         }
         return validKey
     }
+
+    func generateNewKey() {
+        UserDefaults.standard.set(nil, forKey: Constants.indexSecureKeychainPrefix.rawValue)
+        genertateAndSaveKey()
+    }
+
+    #warning("use only for migration purposes") 
+    func getOldKeychainKey() -> Data? {
```

Comment:
The last public version does not use Realm encryption at all. And future versions will always use the same key going forward. Does that not mean that we should not need this method?